frontend/src/app/pages/staff-profile-dashboard/staff-profile-dashboard.component.html

```diff
@@ -0,0 +1,31 @@
+<h1 class="title">Staff Profile Dashboard</h1>
+<div class="profile-dashboard-container">
+    <div class="form">
```

Comment:
This is kind of my fault, but building forms this way using reference identifiers is apparently not the best practice for angular. I changed my form to use angular's reactive form module. Please follow and update.